Issutracker : #i48304#
CWS         : N/A
Author      : NAKATA Maho <maho@openoffice.org> (JCA)
Description : getuid.so is not req'ed for FreeBSD

Index: bin/modules/installer/download.pm
===================================================================
RCS file: /cvs/tools/solenv/bin/modules/installer/download.pm,v
retrieving revision 1.13
diff -u -r1.13 download.pm
--- solenv/bin/modules/installer/download.pm	25 Apr 2005 11:55:33 -0000	1.13
+++ solenv/bin/modules/installer/download.pm	28 Apr 2005 00:11:54 -0000
@@ -189,9 +189,10 @@
 #	my $systemcall = "cd $installdir; tar $ownerstring -cf - * | /usr/bin/sum |";
 
 #	$ENV{'LD_PRELOAD'} = $getuidlibrary;
-#	my $systemcall = "cd $installdir; tar -cf - * | /usr/bin/sum |";
+#getuid.so is req'ed for Linux/Solaris? FreeBSD/MacOSX don't need it.
+	my $systemcall = "cd $installdir; tar -cf - * | /usr/bin/sum |";
 
-	my $systemcall = "cd $installdir; LD_PRELOAD=$getuidlibrary tar -cf - * | /usr/bin/sum |";
+#	my $systemcall = "cd $installdir; LD_PRELOAD=$getuidlibrary tar -cf - * | /usr/bin/sum |";
 
 	my $sumoutput = "";
 
@@ -228,7 +229,8 @@
 	
 	my $getuidlibraryname = "getuid.so";
 	my $getuidlibraryref = installer::scriptitems::get_sourcepath_from_filename_and_includepath(\$getuidlibraryname, $includepatharrayref, 0);
-	if ($$getuidlibraryref eq "") { installer::exiter::exit_program("ERROR: Could not find $getuidlibraryname!", "get_path_for_library"); }
+# ad hock prohibiting error
+#	if ($$getuidlibraryref eq "") { installer::exiter::exit_program("ERROR: Could not find $getuidlibraryname!", "get_path_for_library"); }
 
 	return $$getuidlibraryref;
 }
